glean-core/rlb/src/lib.rs

```diff
@@ -151,6 +151,17 @@ where
     f(&mut lock)
 }
 
+/// Launches a new task on the global dispatch queue with a reference to the Glean singleton.
+pub fn launch_with_glean(callback: impl FnOnce(&Glean) + Send + 'static) {
```

Comment:
These don't need to be `pub` (as that would make them accessible outside of this crate).

```suggestion
fn launch_with_glean(callback: impl FnOnce(&Glean) + Send + 'static) {
```